src/background/providers/cloudflare.ts

```diff
@@ -288,19 +289,23 @@ export class CloudflareProvider implements Provider {
         }
 
         const hasBodyParams = QUALIFIED_BODY_PARAMS.every((param) => {
-            return details.requestBody !== null && param in details.requestBody.formData!;
+            return (
```

Comment:
I think it's better to be more specific what you are checking for. Like check for `null`. This line gives a hint to me that `requestBody` may be boolean.

Reply:
agree, fixed now